### ISSUE TITLE
Request for additional a11y fields

### ISSUE THREAD

**Issue by abbybowman** [GSA/site-scanning#1686]
We would like to use Site Scanning's a11y data to help us triage which sites would likely need major accessibility work (if they remain standalone and don't consolidate) vs. minor fixes. The string/JSON format of the `Accessibility - Violations` field currently makes it very hard for us to extract out a total numerical count of violations across the various axe-core groupings. 

Is there any relatively easy way for GSA to provide an `Accessibility - Violation Count` integer field in addition to the string one with details?

**Comment by gbinal** [comment]
Hi @abbybowman - thanks so much for reaching out.  This is great feedback.  

Just to confirm, in the context of this need, are you using the [details file](https://api.gsa.gov/technology/site-scanning/data/weekly-snapshot-accessibility-details.json) as well?   I know that it doesn't explicitly have that sum total number, but it is a pretty direct way of getting at all of the details someone needs for their site in this regard.  

We've been thinking this idea through, though, and are a bit conflicted.  On the one hand, we want to provide value that you will use.  On the other hand, we try to keep the creation of new columns (and thus an increase in file size and cell count, which starts to become an issue for some) in tension with whether the field is generating new value (as opposed to computing something existing).  If we helped to articulate an excel or google sheets formula that would sum the numbers from the existing field, how much would that allay your need?
